src/helpers/storage.js

```diff
@@ -63,9 +69,7 @@ exports.getLastDirectory = () => {
 
 exports.setDirectories = value => {
     load();
-    if (!data[directoriesKey].includes(value)) {
-        data[directoriesKey].push(value);
-    }
+    data[directoriesKey] = value;
```

Comment:
This was making sure that the directories don't get duplicated. You should check this here: https://github.com/wistcc/git-easy/pull/25/files#diff-f83fb6fa12f4e21578af866d73fdf602R54